test/test_interfaces_stub.py

```diff
@@ -7,6 +7,3 @@ def test_knx(self):
 
     def test_dmx(self):
         import shc.interfaces.dmx  # noqa: F401
-
-    def test_midi(self):
-        import shc.interfaces.midi  # noqa: F401
```

Comment:
Had to remove this one.  Well it is tested in test_midi.py which is skipped in the pipeline.  

Is my current proposal sufficient or do you feel like we need this test here?  Then we rollback to your approach and install `midi` but uninstall `python-rtmidi` which is kind of confusing to me.  Maybe we should split the dependency then.